### PR TITLE
Docs: Add Store metrics

### PR DESCRIPTION
**Pull request by TomiwaAribisala-git**
Docs: Add Store metrics, @douglascamata.

### CODE REVIEW

File: docs/components/store.md [hunk 1]
```diff
@@ -569,3 +569,26 @@ If timeout is set to zero then there is no timeout for fetching and fetching's l
 In order to query series inside blocks from object storage, Store Gateway has to know certain initial info from each block index. In order to achieve so, on startup the Gateway builds an `index-header` for each block and stores it on local disk; such `index-header` is build by downloading specific pieces of original block's index, stored on local disk and then mmaped and used by Store Gateway.
 
 For more information, please refer to the [Binary index-header](../operating/binary-index-header.md) operational guide.
+
+## Metrics
+
+### List of Metrics Exported By Store
```
Comment (fpetkovski) [inline docs/components/store.md:575]:
```suggestion
### List of metrics exported by Bucket Store
```

**Review by fpetkovski — APPROVED** [review]
Real nice

File: docs/components/store.md (hunk 1 above)
Comment (TomiwaAribisala-git) [reply docs/components/store.md:575]: Thanks for the review @fpetkovski; Am I to sign off and commit the suggestion?

Comment (fpetkovski) [reply docs/components/store.md:575]: Sure, that should work.

**Review by douglascamata — APPROVED** [review]

**Pull request closed without merging**